pkg/azure/azureconfig/config.go

```diff
@@ -6,6 +6,15 @@ type Config struct {
 	// string should be loaded.
 	EnvConnectionString string `json:"env"`
 
+	// StorageAccount contains the name of the storage account to be used by the Azure storage plugin, if the azure connection environment variable is not set this and StorageAccountResourceGroup are populated and the user is logged in with the Azure CLI
+	// the Storage Account Key will be looked up at runtime using the logged in users credentials
+	StorageAccount string `json:"account"`
+	// StorageAccountResourceGroup contains the name of the storage account to be used by the Azure storage plugin, if the azure connection environment variable is not set this and StorageAccount are populated and the user is logged in with the Azure CLI
+	// the Storage Account Key will be looked up at runtime using the logged in users credentials
+	StorageAccountResourceGroup string `json:"resourcegroup"`
```

Comment:
nit: Can we update this to be `resourceGroup` and/or the subscription id field below to be `subscriptionid`?  (to match case patterns)